internal/application/command.go

```diff
@@ -33,32 +34,43 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/v3/models"
 )
 
-func GetCommand(ctx context.Context, deviceName string, commandName string, queryParams string, dic *di.Container) (*dtos.Event, errors.EdgeX) {
+func GetCommand(ctx context.Context, deviceName string, commandName string, queryParams string, regexCmd bool, dic *di.Container) (*dtos.Event, errors.EdgeX) {
```

Comment:
I thought we didn't need to know the name was or wasn't a RegEx and always treat it as a RegEx since the actual name is a perfect RegEx match.